Create almalinux.mirror.quickhost.uk.yaml

qhgh opened the pull request:
Adding public mirror for AlmaLinux and Kitten

jonathanspw commented:
Thanks for the mirror!  The config is missing the extra lines required for mirroring kitten.  It also seems that kitten mirroring exists within the main almalinux folder which is not allowed and will cause breakage.

Can you please mirror kitten to another root-level project directory on your mirror and add the config lines for it?

qhgh commented:
Hi Jonathan,

Sorry for the delay! 

I’ve moved kitten to root and added the lines in .yml

```
name: mirror.quickhost.uk
address:
  http: http://mirror.quickhost.uk/almalinux/
address_optional:
  kitten:
    http: http://mirror.quickhost.uk/almalinux-kitten/
update_frequency: 6h
geolocation:
  country: GB
  state_province: London 
  city: London
sponsor: QuickHost
sponsor_url: https://www.quickhost.uk/
email: ***@***.***
```

I hope this helps!

Cerrie 

> On 22 Nov 2024, at 15:09, Jonathan Wright ***@***.***> wrote:
> 
> 
> Thanks for the mirror! The config is missing the extra lines required for mirroring kitten. It also seems that kitten mirroring exists within the main almalinux folder which is not allowed and will cause breakage.
> 
> Can you please mirror kitten to another root-level project directory on your mirror and add the config lines for it?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/AlmaLinux/mirrors/pull/1148#issuecomment-2493975523>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AKNROBKUG4YE72OESYEPOB32B5CLVAVCNFSM6AAAAABSJDVFXWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIOJTHE3TKNJSGM>.
> You are receiving this because you authored the thread.
>

jonathanspw commented:
Thanks for that!

Looks like the mirror hasn't synced since December.  Can you please check your cron, and also adjust it to 3h or less?

jonathanspw commented:
Syncing looks fine now, but can you adjust the frequency to every 3h or less?  This should be adjusted in the yml file as well as the actual sync - anything greater will cause the mirror to not be listed in the mirrorlist as our checkers will see it as outdated.

jonathanspw commented:
Bump.  The 3h change is all that's needed to get this merged (making sure to actually change your sync frequency too).

jonathanspw commented:
@qhgh one final bump.  Only the minor change to the sync frequency is needed to merge this.
